### DISCUSSION TITLE
How to write this way `<component is="template">111</component>`

### DISCUSSION THREAD

**Discussion by Lazydd** [vuejs/rfcs#612] [General]
[demo](https://play.vuejs.org/#eNp9kc1uwjAQhF9l60taCREhblGgf+LQHtqq5ehLapZgcOzI3gASyrt3nQiaA+Jm73yznpFP4rmux/sGRSbyoLyuCQJSU8+l1VXtPMEJPK6hhbV3FSSMJtIqZwODG3dY6gphFpF78g0+SJun/R7ewBfCqjYFId8A8t+GyFl4Ukar3UyKwYa781mKudoUtsQ87fHeqhzHsWgJMh2G1kdIVnqfQAZJfKwwxAnFfDKZ5OnFxDvydJBFjAQFbrHW5XgbnOX6p/iMFNGiDfrPmjS3lCKDTolaYYw7vHezWHZ0nqsNqt2V+TYc40yKL48B/Z67XTQqfInUy4ufDzzy+SJWbtUYpm+I3xicaWLGHntp7IpjD7gu7Vv3idqWy7A4EtpwLhWDRrLteCn4Y19vVP+POx1PO5+0rWj/AFGvwHw=)

**Comment by posva** [comment]
template is special HTML tag, that's why you will see nothing (after fixing the typo in the playground)
Remember to use the [Discord chat](https://chat.vuejs.org/) to ask questions or Discussions (if enabled)!
